dist/listbox-button/listbox-button.css

```diff
@@ -56,6 +56,20 @@ button.expand-btn[aria-expanded="true"] ~ div.listbox-button__listbox,
 .listbox-button button.btn--borderless[aria-expanded="true"] ~ .listbox-button__listbox {
   top: 41px;
 }
+.listbox-button.listbox-button--form button {
+  background-color: var(--listbox-button-background-color, var(--color-background-secondary));
+  border-color: var(--listbox-button-border-color, var(--color-stroke-default));
+}
+.listbox-button.listbox-button--form button[disabled],
+.listbox-button.listbox-button--form button[aria-disabled="true"] {
+  border-color: var(--listbox-button-disabled-border-color, var(--color-background-disabled));
+  color: var(--listbox-button-disabled-foreground-color, var(--color-foreground-disabled));
+}
+.listbox-button.listbox-button--form button:focus {
+  background-color: var(--combobox-textbox-focus-background-color, var(--color-background-primary));
+  border-color: var(--combobox-textbox-focus-border-color, var(--color-foreground-primary));
+  outline: 0;
```

Comment:
Hmm... since `combobox` was one of the components mentioned in the issue, I was actually basing it off `combobox` which looks like this when focused: 
<img width="335" alt="image" src="https://user-images.githubusercontent.com/1675667/212981443-98423834-f71f-4276-bb9a-2ce54caad407.png">

Ok, I guess I'll pattern it after `select` instead.

Reply:
Ok, I've updated the focus to match `select`.